infra/src/migration/1646683871219-Initial.js

```diff
@@ -0,0 +1,194 @@
+    // security group
+    await queryRunner.query(`
+      BEGIN;
+        INSERT INTO aws_security_group (description, group_name)
+        VALUES ('${PROJECT_NAME} security group', '${SECURITY_GROUP}');
+
+        INSERT INTO aws_security_group_rule (is_egress, ip_protocol, from_port, to_port, cidr_ipv4, description, security_group_id)
+        SELECT false, 'tcp', ${PORT}, ${PORT}, '0.0.0.0/0', '${SECURITY_GROUP}', id
+        FROM aws_security_group
+        WHERE group_name = '${SECURITY_GROUP}';
+
+        INSERT INTO aws_security_group_rule (is_egress, ip_protocol, from_port, to_port, cidr_ipv4, description, security_group_id)
+        SELECT true, 'tcp', 443, 443, '0.0.0.0/0', '${SECURITY_GROUP}', id
+        FROM aws_security_group
+        WHERE group_name = '${SECURITY_GROUP}';
+      COMMIT;
+    `);
+
+    // load balancer + cloudwatch
+    await queryRunner.query(`
+      BEGIN;
+        INSERT INTO aws_target_group
+            (target_group_name, target_type, protocol, port, vpc, health_check_path)
+        VALUES
+            ('${TARGET_GROUP}', 'ip', 'HTTP', ${PORT}, 'default', '/health');
+
+        INSERT INTO aws_load_balancer
+            (load_balancer_name, scheme, vpc, load_balancer_type, ip_address_type)
+        VALUES
+            ('${LOAD_BALANCER}', 'internet-facing', 'default', 'application', 'ipv4');
+
+        INSERT INTO aws_load_balancer_security_groups
+            (aws_load_balancer_id, aws_security_group_id)
+        VALUES
+            ((SELECT id FROM aws_load_balancer WHERE load_balancer_name = '${LOAD_BALANCER}' LIMIT 1),
+              (SELECT id FROM aws_security_group WHERE group_name = '${SECURITY_GROUP}' LIMIT 1));
+
+        INSERT INTO aws_listener
+            (aws_load_balancer_id, port, protocol, action_type, target_group_id)
+        VALUES
+            ((SELECT id FROM aws_load_balancer WHERE load_balancer_name = '${LOAD_BALANCER}' LIMIT 1),
+              ${PORT}, 'HTTP', 'forward', (SELECT id FROM aws_target_group WHERE target_group_name = '${TARGET_GROUP}' LIMIT 1));
+
+        INSERT INTO log_group (log_group_name)
+        VALUES ('${LOG_GROUP}');
+      COMMIT;
+    `);
+
+    // container (ECR + ECS)
+    await queryRunner.query(`
+      BEGIN;
+        INSERT INTO aws_public_repository (repository_name) VALUES ('${REPOSITORY}');
+
+        INSERT INTO aws_cluster (cluster_name) VALUES('${CLUSTER}');
+
+        INSERT INTO aws_task_definition ("family", cpu_memory)
+        VALUES ('${TASK_DEF_FAMILY}', '${TASK_DEF_RESOURCES}');
+
+        INSERT INTO aws_container_definition ("name", public_repository_id, task_definition_id, log_group_id, tag, essential, memory_reservation, host_port, container_port, protocol)
+        VALUES (
+          '${CONTAINER}',
+          (select id from aws_public_repository where repository_name = '${REPOSITORY}' limit 1),
+          (select id from aws_task_definition where family = '${TASK_DEF_REPO_FAMILY}' and status is null limit 1),
```

Comment:
Here you are using 2 different task definitions families. You need to stick to one of them.

Reply:
I'd replace `TASK_DEF_REPO_FAMILY` with `TASK_DEF_FAMILY` all over the migration.

Reply:
that was it, thank you!